Assets/RGLUnityPlugin/Scripts/LowLevelWrappers/RGLNativeAPI.cs

```diff
@@ -182,6 +182,17 @@ public static extern int rgl_node_points_simulate_snow(ref IntPtr node, float mi
         public static extern int rgl_node_points_simulate_snow_configure_defaults(IntPtr node, int snowflakes_id, float full_beam_intensity,
             float snowflakes_laser_retro);
 
+        [DllImport("RobotecGPULidar")]
+        public static extern int rgl_node_points_simulate_rain(ref IntPtr node, float min_range, float max_range, float rain_rate,
+            Int32 num_channels, float beam_divergence , Int32 numerical_threshold, float occupancy_threshold);
+
+        [DllImport("RobotecGPULidar")]
+        public static extern int rgl_node_points_simulate_rain_configure_defaults(IntPtr node, int droplets_id, float full_beam_intensity,
+            float droplets_laser_retro);
+
+        [DllImport("RobotecGPULidar")]
+        public static extern int rgl_node_points_simulate_fog(ref IntPtr node, float attenuationCoefficient, float r1, float r2);
```

Comment:
`rgl_node_points_simulate_rain_configure_defaults` call is not imported - please add it like is is done for snow and rain. Note that there will be an additional property - FogId - in FogManager.